### DISCUSSION TITLE
Missing required fields

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [stevearc/overseer.nvim#289] [Q&A]
i'm following the tutorials, but the LUA LSP server throws the following warning:
```
Missing required fields in type `overseer.Config`: `default_template_prompt`, `log` Lua Diagnostics. (missing-fields) [3, 27]
```

Why is that?

![image](https://github.com/stevearc/overseer.nvim/assets/54762742/baf299cd-723b-4f9f-bd65-7cb28224b865)

**Comment by stevearc** [comment]
This was just an out of date type definition. Should be fixed now
